### DISCUSSION TITLE
신규 프로젝트로 이슈 정리

### DISCUSSION THREAD

**Discussion by EightonEight** [KoreanGRF/KoreanTrainSet#293] [건의(Suggestions)]
#292 와 같은 정리용 이슈를 별도로 올리는 것보단 신규 프로젝트로 관리하는 것이 좋을 것 같습니다.

**Comment by EightonEight** [comment]
~[새로운 프로젝트](https://github.com/KoreanGRF/KoreanTrainSet/projects/4)가 생성되었으므로 해당 건의는 닫아도 됩니다.~

**Reply by telk5093** [reply]
건의는 이슈와 달라서 Close한다는 개념이 없습니다 😅
